package.json

```diff
@@ -14,22 +14,27 @@
   "scripts": {
     "dev": "pnpm install && pnpm i18n:create-locales-list && pnpm i18n:update-locales && cross-env nuxt --hostname 0.0.0.0",
     "build": "pnpm i18n:get-translations && nuxt build",
+    "build-only": "nuxt build",
     "generate": "nuxt generate",
     "start": "nuxt start",
+    "prod": "pnpm build-only && pnpm start",
     "storybook": "nuxt storybook",
     "test": "jest",
-    "test:all": "pnpm e2e:ci & jest",
+    "test:all": "pnpm test:e2e & jest",
     "test:watch": "jest --collectCoverage=false --watch",
-    "test:e2e": "playwright test",
-    "test:e2e-debug": "PWDEBUG=1 playwright test",
-    "build-and-e2e": "start-server-and-test 'cross-env API_URL=http://localhost:3000 nuxt build && pnpm start' http://localhost:8443 test:e2e",
+    "talkback": "node ./test/proxy.js",
+    "test:e2e": "cross-env PLAYWRIGHT_VERSION=1.17.1 TEST_TYPE=e2e docker-compose -f docker-compose.playwright.yml up --abort-on-container-exit --exit-code-from playwright",
+    "test:e2e:local": "playwright test -c test/e2e",
```

Comment:
I didn't know we can have double `:` in the script name 👍

Reply:
Yeah! I've used [this package (`scripty`)](https://www.npmjs.com/package/scripty) in the past when I worked on the [debtcollective/dispute-tools](https://github.com/debtcollective/dispute-tools) and it uses `:` as the path delimiter. It lets you define separate scripts for windows and unix-types and reference them by the same name. Very helpful! If we don't switch to `just` we could use `scripty` to more easily create cross-platform scripts for the frontend.